.github/workflows/ci.yml

```diff
@@ -33,47 +33,47 @@ jobs:
         include:
         - name: Node.js 0.10
           node-version: "0.10"
-          npm-i: mocha@3.5.3 nyc@10.3.2 supertest@2.0.0
+          npm-i: mocha@3.5.3 nyc@10.3.2 semver@6.3.0 supertest@2.0.0
```

Comment:
@dougwilson This is no longer needed.

Reply:
You're right. Do you want to remove or have me remove?

Reply:
I'll remove it for us.